modules/interactive_music/doc_classes/AudioStreamPlaybackPlaylist.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AudioStreamPlaybackPlaylist" inherits="AudioStreamPlayback" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+	</brief_description>
```

Comment:
It would be nice to have at least a basic brief description, like referring to AudioStreamPlaylist.
Same below.

Not sure about full description, you could leave it out if you don't feel like writing it.